src/createServer.js

```diff
@@ -1,3 +1,50 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+'use strict';
+
+const http = require('http');
+const { getParamsOfUrl } = require('./getParamsOfUrl');
+
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const [text, toCase] = getParamsOfUrl(req);
+
+    const errors = [];
+
+    /* eslint-disable */
+    if (!text) {
+      errors.push({ message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
+    }
+
+    if (!toCase) {
+      errors.push({ message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
+    } else if (!['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'].includes(toCase)) {
+      errors.push({ message: `This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.` });
+    }
+    /* eslint-enable */
```

Comment:
Let's decompose your logic
Move it to the separate function
```suggestion
    const errors = ...();
```